### ISSUE TITLE
Make the navbar responsive

### ISSUE THREAD

**Issue by mugisha23** [burundiarxiv/open-library#30]
I think we should make the navbar looks better on small screens. 
![navbar](https://user-images.githubusercontent.com/20721985/103762506-12885700-5021-11eb-870f-183604fc1abd.PNG)

I don't have a clear idea on how it should look like but i think there's something to improve. @lionel-k @muco-rolle what do you think

**Comment by lionel-k** [comment]
I agree we should work on it.

We could have something like this:
https://www.w3schools.com/howto/howto_js_mobile_navbar.asp
Let's implement the vertical one. What do you think ?

[image: Screenshot 2021-01-06 at 12.28.50.png]

**Comment by lionel-k** [comment]
![Uploading Screenshot 2021-01-06 at 12.30.05.png…]()

**Comment by mugisha23** [comment]
yeah, I also prefer the vertical one. I think it's fine if I start to work on it ?

**Comment by mugisha23** [comment]
I don't know what happened but I'm unable to see the screenshot you sent.

**Comment by lionel-k** [comment]
Yes. That would be great

**Comment by mugisha23** [comment]
I'm strugling to make the navbar responsive, I'm may need some guidance. I managed to make the toggle appear on the screen but have 2 problems:
- the toggle appear on all screen size even tough i used the media query
- I also don't see how i can put those navbar element into the toggle. Also can't find a way to setup a function that wll handle the event
.I used the ressource you gave me, also did some search on google but it still not working.
Any kind of help is welcome

**Comment by lionel-k** [comment]
Can you please open a Pull Request with the code you already have and I will take a look and see how I can guide you..
